docker/Dockerfile_i686

```diff
@@ -50,7 +50,7 @@ RUN yum install autoconf automake bzip2 cmake freetype-devel gcc gcc-c++ libtool
 	tar xjvf ffmpeg-snapshot.tar.bz2 && \
 	cd ffmpeg && \
 	PATH=~/bin:$PATH && \
-	PKG_CONFIG_PATH="$HOME/ffmpeg_build/lib/pkgconfig" ./configure --prefix="$HOME/ffmpeg_build" --extra-cflags="-I$HOME/ffmpeg_build/include" --extra-ldflags="-L$HOME/ffmpeg_build/lib" --enable-libvpx --enable-shared --enable-pic --bindir="$HOME/bin" && \
+	PKG_CONFIG_PATH="$HOME/ffmpeg_build/lib/pkgconfig" ./configure --prefix="$HOME/ffmpeg_build" --extra-cflags="-I$HOME/ffmpeg_build/include" --extra-ldflags="-L$HOME/ffmpeg_build/lib" --enable-libvpx --enable-openssl --enable-protocol=file,ftp,http,https,httpproxy,hls,mmsh,mmst,pipe,rtmp,rtmps,rtmpt,rtmpts,rtp,sctp,srtp,tcp,udp --enable-shared --enable-pic --bindir="$HOME/bin" && \
```

Comment:
Is there a particular reason to enable these specific protocols? By default, all supported protocols are enabled AFAICS.

Reply:
(Not criticizing; just wondering since this looks like just a copied full list but it's not actually the full list.)

Reply:
> Is there a particular reason to enable these specific protocols? By default, all supported protocols are enabled AFAICS.

@native-api Thanks for pointing this out. No there is no specific reason, I just thought these are the most commonly used protocols. Should I remove them?

Reply:
Yes. I don't think we should change the default without a real reason (I'm not implying that there's none, just that none was presented).